packages/vue-client/src/components/boards/BadukBoardAbstract.vue

```diff
@@ -1,15 +1,14 @@
 <script setup lang="ts">
 import { Color } from "@ogfcommunity/variants-shared";
-import { PolygonalBoardHelperFunctions } from "@ogfcommunity/variants-shared/src/baduk/board/PolygonalBoardHelper";
+import type {
+  BadukWithAbstractBoardConfig,
+  BadukWithAbstractBoardState,
+} from "@ogfcommunity/variants-shared/src/badukWithAbstractBoard/badukWithAbstractBoard";
+import { propsToAttrMap } from "@vue/shared";
```

Comment:
I think you can delete this line to fix the lint error.

Reply:
Right, removing the unused import fixed the linter error. Thanks for the tip!

At some point I ran _yarn lint_, hence the white space changes in the polygonal helper. Afterwards it showed me a lot of files in the changes. But when I opened some as Diff, I did not find any changes. I found that weird and decided to only stage and commit files that I changed.

Reply:
Oh hmm that is strange. I wonder if it could be some whitespace strangeness or something 🤷‍♂️ anyway thanks for being vigilant about the diff!